Neos.Flow/Classes/Persistence/Doctrine/Query.php

```diff
@@ -12,15 +12,18 @@
  */
 
 use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\Exception as DbalException;
```

Comment:
I would actually prefer to do DbalException everywhere rather than the opposite, because it fits more into our naming schema? Before it was set because the class name contained DBAL.

Reply:
But maybe do this in a separate "just cosmetic" PR to keep this one as small as possible?